App.tsx

```diff
@@ -1,11 +1,13 @@
 import {StyleSheet, SafeAreaView} from 'react-native';
 import React from 'react';
 import {ToDoList} from './src/screens/toDoList';
+import FlatListToDoList from './src/screens/toDoList/FlatListToDoList';
```

Comment:
```suggestion
import {ToDoList, FlatListToDoList} from './src/screens/toDoList';
```
export FlatListToDoList from index.ts